apps/desktop-wallet/src/storage/addresses/addressesSelectors.ts

```diff
@@ -85,15 +82,34 @@ export const makeSelectAddressesAlphAsset = () =>
 
 export const makeSelectAddressesTokens = () =>
   createSelector(
-    [selectAllFungibleTokens, selectAllNFTs, makeSelectAddressesAlphAsset(), makeSelectAddresses(), selectAllPrices],
-    (fungibleTokens, nfts, alphAsset, addresses, tokenPrices): Asset[] => {
-      const tokenBalances = getAddressesTokenBalances(addresses)
-      const tokens = calculateAssetsData([alphAsset, ...tokenBalances], fungibleTokens, nfts, tokenPrices)
-
-      return sortAssets(tokens)
```

Comment:
This is where I remove the sorting (which comes from the shared package, which relies on the priceSlice.